bqetl

```diff
@@ -17,27 +17,13 @@ if ! command -v python3 &> /dev/null; then
 fi
 
 if [ "$CMD" == "bootstrap" ]; then
-    if [ -d "venv" ]; then
-        echo 'venv subdirectory already exists, execute "rm -rf venv" and re-run this command if you want to recreate it'
-        exit 1
-    else
-        python3 -m venv venv
+    if [[ "$2" == "--with-java" ]]; then
+      mvn package
     fi
-
-    # set this so we don't attempt to recompile pandas and numpy on more recent versions of Mac
-    # may be unnecessary once Mac OS X 11 wheels become commonplace
-    # see e.g. https://github.com/scipy/scipy/issues/13102#issuecomment-788160041
-    export SYSTEM_VERSION_COMPAT=1
-
-    venv/bin/pip install --no-deps -r requirements.txt
-    venv/bin/pip install -e .
+    poetry install
     echo "bqetl configured! It should now be ready for use."
     exit 0
 fi
 
-if [ ! -d "venv" ]; then
-    echo "Please run ./bqetl bootstrap"
-    exit 1
-fi
-
-venv/bin/bqetl "$@"
+export CLOUDSDK_PYTHON=$(which python)
+poetry run bqetl "$@"
```

Comment:
script/bqetl needs to be updated accordingly

Reply:
prefixing everything with `poetry run` will be an issue inside the docker container since poetry should not be in the final layer.

Reply:
The reason I only prefix this command with `poetry run` is because (I believe) this is supposed to be the main developer interface. For scripts in the `scripts/*` directory I explicitly did not add the `poetry run` prefix because the context could be within a container.

`poetry run` just runs the command in a subshell with the virtual environment activated. if a dev wanted to run a script in `scripts/` they could do so by either prefixing with `poetry run` or activating the virtual env (`poetry shell` will do that